src/checkout/checkout-store-selector.ts

```diff
@@ -49,6 +50,13 @@ export default interface CheckoutStoreSelector {
      */
     getConfig(): StoreConfig | undefined;
 
+    /**
+     * Gets the sign-in email.
+     *
+     * @returns The sign-in email object if sent, otherwise undefined
+     */
+    getSignInEmail(): SignInEmail | undefined;
```

Comment:
The name is a little awkward but I can't come up with anything better so I guess this is good enough.

Reply:
I know.. _naming is hard_